spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -84,52 +63,5 @@
         expect(response.body).to match 'can&#39;t be blank'
       end
     end
-
-    context 'KBV off' do
```

Comment:
See my commit message. These tests were already covered by feature specs, except for one of them, which I added to the question cycle feature specs. Integration tests don't belong in controller tests.